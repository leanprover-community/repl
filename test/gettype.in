{"decl": "theorem show_p (p : Prop) (h : p) : p := by exact h"}

{"decl": "def show_q (p : Prop) (h : p) : p := h"}

{"decl": "def show_q (p : Prop) (h : p) : p := h\n\ndef show_p (q : Prop) (h : q) : q := h"}

{"decl": "variable (q : Prop)\ndef show_q (p : Type) (h : q) : q := h"}
