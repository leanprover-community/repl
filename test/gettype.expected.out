{"types": ["∀ (p : Prop), p → p"]}

{"types": ["∀ (p : Prop), p → p"]}

{"types": ["∀ (p : Prop), p → p", "∀ (q : Prop), q → q"]}

